lib/org/chromium/webidl/FetchSpecRegistrySelector.js

```diff
@@ -0,0 +1,54 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+'use strict';
+
+foam.CLASS({
+  package: 'org.chromium.webidl',
+  name: 'FetchSpecRegistrySelector',
+  implements: [
+    'foam.box.RegistrySelector',
+    'foam.core.StubFactory',
+  ],
+
+  requires: [
+    'foam.box.BoxRegistry',
+    'foam.box.Context',
+    'foam.box.node.ForkBox',
+    'foam.core.StubFactorySingleton',
+  ],
+  imports: ['registry'],
```

Comment:
Doesn't look like this is being used (anymore). Leftover from iterations of reuse/don't-reuse-existing-registry attempts. Sorry.